src/base-input.js

```diff
@@ -502,6 +515,8 @@ export default class BaseInput extends LitElement {
       return !this.placeholder ? 'MM/DD/YYYY' : this.placeholder;
     } else if (this.type === 'month-year') {
       return !this.placeholder ? 'MM/YY' : this.placeholder;
+    } else if (this.type === 'month-fullYear') {
+      return !this.placeholder ? 'MM/YYYY' : this.placeholder;
```

Comment:
Please add the following 

```js
else if (this.type === 'month-fullYear') {
  return true;
}
```

To the `defineInputIcon()` and `defineLabelPadding()` functions.